Automattic/newspack-multibranded-site
# feat: add auto updater

Implements an auto-updater based on GitHub releases.

Closes #30 

### How to test

1. Check out this branch, and modify line 5 of the `newspack-multibranded-site.php` file to a lower version (0.9.0)
2. Visit the Dashboard -> Updates page and confirm you see an update available for the plugin
3. Update and confirm the plugin files are replaced with the latest available version from GitHub
